File: labs-training-master/java/solid/solid-java-1/src/main/java/es/emergya/labs/training/solid_java_1/srp/excercise_00/Car.java
package es.emergya.labs.training.solid_java_1.srp.excercise_00;

public class Car {
	
	private final String id;
	private final String model;
	private final String brand;

	public Car(String id, String model, String brand) {
		
		this.id = id;
		this.model = model;
		this.brand = brand;
	}

	public String getId() {
		return id;
	}

	public String getModel() {
		return model;
	}

	public String getBrand() {
		return brand;
	}
}

File: labs-training-master/SolidPythonResuelto/main/srp/Car.py
class Car:
    def __init__(self, id : str,  model : str,  brand : str) :
        self.id = id
        self.model = model
        self.brand = brand
    def  getId(self) :
        return  self.id
    def  getModel(self) :
        return  self.model
    def  getBrand(self) :
        return  self.brand